src/BugsnagUnity/Native/Android/NativeClient.cs

```diff
@@ -132,6 +112,28 @@ public LastRunInfo GetLastRunInfo()
             return NativeInterface.GetlastRunInfo();
         }
 
+        public void ClearNativeMetadata(string section)
+        {
+            NativeInterface.ClearMetadata(section);
+        }
+
+        public void ClearNativeMetadata(string section, string key)
+        {
+            NativeInterface.ClearMetadata(section,key);
+        }
+
+        public IDictionary<string, object> GetNativeMetadata()
+        {
+            return NativeInterface.GetMetadata();
+        }
+
+        public void SetNativeMetadata(string section, IDictionary<string, object> data)
```

Comment:
I'm not sure if this function makes sense - I would assume that this clears any existing metadata, then adds all the key-value pairs in `data`. Right now the implementation just adds metadata, so it feels like the values could be wrong if someone forgot to call `ClearNativeMetadata` first?

Reply:
The behaviour is correct, but the name was missleading, i have renamed it 👍